.env

```
@@ -0,0 +1,4 @@
+   PRIVATE_KEY=011d4ba8e7b549ee9cd718df4c25abed
```

dotenv-linter (3.3.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)